cmd/query/app/flags.go

```diff
@@ -70,7 +70,8 @@ type QueryOptionsStaticAssets struct {
 // QueryOptionsBase holds configuration for query service shared with jaeger(v2)
 type QueryOptionsBase struct {
 	// BasePath is the base path for all HTTP routes
-	BasePath string
+	BasePath   string
+	UIBasePath string
```

Comment:
given the amount of confusion this is already causing, it's a good idea to provide clear documentation for these fields